components/ca-import.css

```diff
@@ -0,0 +1,42 @@
+ca-import { position: relative; }
+
+/* fake upload button */
+ca-import form { display: inline-block; position: relative; overflow: hidden; color: #fff; cursor: pointer; background-color: #333; border: 1px solid #333; margin: 10px 0; padding: 0.5em 2em; border-radius: 5px; text-align: center; }
```

Comment:
why are these all on one line??  please can you refactor these into the normal format, for consistency and readability?